README.md

```diff
@@ -22,13 +22,13 @@ All information below is for developers.
 
 Install [Node](https://nodejs.org/en/download/) and [Rust](https://doc.rust-lang.org/cargo/getting-started/installation.html).
 
-To test the viewer, you need to build the viewer and then the serve script will handle dependencies for examples:
+To test the viewer, you need to build the viewer and then start the examples. 
 
 ```
 cd viewer
-yarn install
-yarn build
-yarn serve
+yarn && yarn run build
+cd ../examples
+yarn && yarn run start
```

Comment:
Not something for now, but could maybe examples be included as workspaces as well?

Reply:
yes, but not in scope for this PR